server/api/api/endpoints/nuclio.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2023 Iguazio
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+from fastapi import APIRouter, Depends
+
+import mlrun.common.schemas
+import server.api.utils.clients.async_nuclio
+from server.api.api import deps
+
+router = APIRouter()
+
+
+@router.get("/projects/{project}/nuclio/api-gateways")
+async def list_api_gateways(
+    project: str,
+    auth_info: mlrun.common.schemas.AuthInfo = Depends(deps.authenticate_request),
+):
+    await server.api.utils.auth.verifier.AuthVerifier().query_project_permissions(
+        project_name=project,
```

Comment:
in `httpdb` you allow `project` to be None - will it be authorized in that case?
MLRun is very project oriented, and we done want users to see resources for all project.
Perhaps an enrichment will be a good idea:
```
project = project or config.default_project
```
this is very common in the API ^